src/assets/styles/app/_layout.scss

```diff
@@ -360,6 +360,33 @@ input[type="checkbox"] {
   }
 }
 
+// Radio
+// ---------------------------------
+input[type="radio"] {
+  $radio-size:     16px;
+
+  position: relative;
```

Comment:
very similar to the checkbox code but wanted the radio buttons to be pretty "on-brand". There's probably a better way to do this 🤷‍♂️